recipes/arvados-python-client/meta.yaml

```diff
@@ -10,33 +10,29 @@ source:
 
 build:
   noarch: python
-  number: 0
+  number: 1
 
 requirements:
   host:
     - python >=3
     - setuptools
```

Comment:
to you need to restrict/pin setuptools here as well?

Reply:
I have applied the same constraints on setuptools in the host section as in the run section.

Not sure if this is necessary, but can't hurt I guess.